README.md

```diff
@@ -124,7 +129,7 @@ You can set the BPMN content using one of the following ways:
 ```
 
 ### 👤 User documentation
-The User documentation with the feature list & the public API is stored on [Github pages](https://process-analytics.github.io/bpmn-visualization-js/).
+The User documentation with the feature list & the public API are available in the [documentation site](https://process-analytics.github.io/bpmn-visualization-js/).
```

Comment:
It's the User doc which is available on this link.
There is another link in the User doc for the public API 😉 

```suggestion
The User documentation (with the feature list & the public API) is available in the [documentation site](https://process-analytics.github.io/bpmn-visualization-js/).
```

Reply:
:+1 this is much clearer like this with details between paranthesis.